content/raw/chat/activity_center.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "activity center spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Activity Center
+
+## Designs
+
+- [designs](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5358%3A5666)
+- [updated designs](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1466%3A343105)
+
+## Use Cases
+
+TODO
+
+## Functional Requirements
+
+### General
+
+* The Activity Center MUST be displayed everywhere in the application
+* The Activity Center MUST support filtering by 5 categories: "All", "Mentions", "Replies", "Contact Requests", "Identity Checks"
+* The Activity Center MAY contain the following type of items:
```

Comment:
According new designs there should be those items:
- Join request (admin)
Actions: link to community, accept, decline
Categories: Admin, Membership (?)

- Mentions
Types: mention in a community, mention in public chat, mention in a group, mention in a 1:1
Actions: link to mention in chat
Categories: Mentions

- Replies
Actions: link to reply in chat
Categories: Replies

- Contact request (incoming)
Features: contact request message
Actions: accept, decline
Categories: Contact Requests

- Contact request (outgoing)
Features: contact request message, state (pending, accepted, declined)
Categories: Contact Requests
(?) How to reject contact request?

- Group chat invite
Features: group preview
Actions: link to group chat, accept, decline
Categories: Membership

- Added to group chat
Actions: link to group chat
Categories: Membership

- Removed from group chat
Actions: link to group chat
Categories: Membership

- Invite to community
Actions: link to community
Categories: Membership

- Join request (user)
Actions: link to community
Categories: Membership

- Join community (or channel in community)
Actions: link to community/channel
Categories: Membership

- (?) Removed from community?

Reply:
Looks good to me